src/hotspot/os/posix/os_posix.cpp

```diff
@@ -1211,28 +1158,13 @@ void os::Posix::init(void) {
 }
 
 void os::Posix::init_2(void) {
-  log_info(os)("Use of CLOCK_MONOTONIC is%s supported",
-               (_clock_gettime != NULL ? "" : " not"));
+  log_info(os)("Use of CLOCK_MONOTONIC is supported");
```

Comment:
We are keeping this output to avoid breaking whomever might be looking for this text?